Base/res/ladybird/inspector.css

```diff
@@ -271,3 +271,58 @@ details > :not(:first-child) {
     padding-left: 10px;
     padding-right: 10px;
 }
+
+.console-log-table {
+    width: 100%;
+    padding: 0 10px;
+    box-sizing: border-box;
+}
+
+.console-log-table table {
+    width: 100%;
+    table-layout: fixed;
+    border-collapse: collapse;
+    border: 1px solid gray;
+}
+
+.console-log-table thead {
+    border-bottom: 1px solid gray;
+}
+
+.console-log-table th {
+    position: sticky;
+    top: 0px;
+    border: 1px solid gray;
+}
+
+.console-log-table td {
+    border-left: 1px solid gray;
+    border-right: 1px solid gray;
+}
+
+
+@media (prefers-color-scheme: light) {
+    .console-log-table tbody tr:nth-of-type(2n+1) {
+        background-color: rgb(229, 229, 229);
+    }
+
+    .console-log-table tbody tr:hover {
+        background-color: rgb(199, 198, 198);
+    }
+}
+
+@media (prefers-color-scheme: dark) {
+    .console-log-table tbody tr:nth-of-type(2n+1) {
+        background-color: rgb(57, 57, 57);
+    }
+
+    .console-log-table tbody tr:hover {
+        background-color: rgb(80, 79, 79)
+    }
+}
```

Comment:
This file was recently changed to use a single `@media` rule which then sets color variables, please do that here too.

Reply:
Moved all colors to variables.